shortcuts/shortcuts_win.go

```diff
@@ -5,5 +5,5 @@ package shortcuts
 import tcell "github.com/gdamore/tcell/v2"
 
 func addDeleteLeftShortcut() *Shortcut {
-	return addShortcut("delete_left", "Delete left", multilineTextInput, tcell.NewEventKey(tcell.KeyBackspace, rune(tcell.KeyBackspace), tcell.ModNone))
+	return addShortcut("delete_left", "Delete left", multilineTextInput, tcell.NewEventKey(tcell.KeyBackspace, rune(tcell.KeyBackspace), tcell.ModNone),nil)
```

Comment:
Why is the last parameter `nil`? Does this need explanation or do we need to change it?

Reply:
Windows build for some reason wont compile, it does not find anything declared inside shortcuts for some reason, so I can not use shortcuts.addVim or NullVimEvent. It's pretty much a HACK to make it pass the build.